### ISSUE TITLE
please include generated css files in theme

### ISSUE THREAD

**Issue by arikast** [luizdepra/hugo-coder#399]
Great theme thanks!  One request: could you include the generated css files with the theme?  Otherwise I have to setup my own sass compilation, which means that the dark theme does not work "out of the box"

**Comment by luizdepra** [comment]
They are all [here](https://github.com/luizdepra/hugo-coder/tree/master/resources/_gen/assets/scss/scss).
But, they might be outdate. I will check it later.

**Comment by arikast** [comment]
Oh maybe I'm mis-using it then.  I'm new to hugo, and also not generally a front-end expert.  I did a fresh install of hugo followed by a fresh install of this theme, and out of the box the theme looked great except the dark toggle did not work.  

So in troubleshooting I looked for files ending in .css in the theme folder and did not find any.  But looking at the [demo](https://themes.gohugo.io/theme/hugo-coder/) I saw that there is a .css file, which I guessed was generated from the .scss files.  

So that's why I thought maybe I have to manually generate the css from those scss files, and I thought it would be better if you just checked in the generated css.  Is my understanding incorrect?  Is there a way for it to work out of the box without any need for me to manually process the scss files, using just standard hugo (not extended hugo, and not with additional tools like an external scss processor)?
